lite/tests/classification/test_accuracy.py

```diff
@@ -53,18 +53,12 @@ def test_accuracy_computation():
     )
 
     # score threshold, label, count metric
-    assert accuracy.shape == (2, 4)
+    assert accuracy.shape == (2,)
 
     # score >= 0.25
-    assert accuracy[0][0] == 2 / 3
-    assert accuracy[0][1] == 1.0
-    assert accuracy[0][2] == 2 / 3
-    assert accuracy[0][3] == 1.0
+    assert accuracy[0] == 2 / 3
```

Comment:
you should use 
`math.isclose()`

Reply:
I understand that this would be best practive.

However, this will require changing a lot of testing since metrics are dense dictionaries and it has not been a problem up to this point.